rfcs/038-sierra-series/README.md

```diff
@@ -0,0 +1,432 @@
+to mint a canonical identifier in the pipeline and to expose that identifier in the API. 
+Finally, the proposal is to model a series statement or a Host Item Entry with a type `Work`.
+
+### Examples
+ 
+- [1074785](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1074785?lang=eng&suite=cobalt&marcData=Y) 
+This is an example of a series statement in 490 + 830. In this example the information contained in 830 and 490 is 
+  exactly the same, so the result is just one statement. 
+```yaml
+"partOf": [
+  {
+    "title": "Morphogenesis of the vertebrate brain; 2",
+    "workType": {
+      "id": "???",
+      "label": "Volumes",
+      "type": "Format"             
+    },
+    "partOf": [
+      {
+        "title": "Morphogenesis of the vertebrate brain",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  }
+]
+```
+- [3109805](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3109805?lang=eng&suite=cobalt&marcData=Y)
+  490 + 773 with slightly different information
+```yaml
+"partOf": [
+  {   //490
+    "title": "Early European Books : Printed sources to 170; Collection 4",
+    "workType": {
+      "id": "???",
+      "label": "Volumes",
+      "type": "Format"
+    },
+    "partOf": [
+      {
+        "title": "Early European Books : Printed sources to 170",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {   //773
+    "title": "Early European Books",
+    "type": "Work"
+  }
+]
+```
+
+- [3178769](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3178769?lang=eng&suite=cobalt&marcData=Y) 490 + 830 multiple entries + 773
+```yaml
+"partOf": [
+  {   // 490 & 830 with same information
+    "title": "Perspectives in Continental philosophy series; no. 39",
+    "workType": {
+      "id": "???",
+      "label": "Volumes",
+      "type": "Format"             
+    },
+    "partOf": [
+      {
+        "id": "abcdefg",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+              },
+            "value": "1089-3938",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Perspectives in Continental philosophy series",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {  // Second 830 with slighly different information
+    "title": "Fordham perspectives in continental philosophy",
+    "type": "Series"
+  },
+ {   // 830 & 773 contain the same information
+    "title": "ACLS Humanities E-Book.",
+    "type": "Work" //or Series?
+  }
+]
+```
+- [2125597](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2125597?lang=eng&suite=cobalt&marcData=Y)
+  490 with ISSN + 830 and 773. There are 1037 bibs that have the same ISSN in a series statement but
+  with different volume subfield.
+```yaml
+"partOf": [
+  {   // 490 +830 contain duplicated info
+    "partOf": [
```

Comment:
Is there an unintended extra level here?

Reply:
yup, sorry my bad, will fix